src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
OK, I suggest to continue the context discussion in #532. But how to proceed with link and divider? This Dropdown proposal is a result of the AppointmentItem and Cal Revision. I think we might need JS signals on the items quickly and working with the original action array only, won't give us this in my understanding.

On the other hand we do not have a Link or Divider ready. Should we try to provide this? Would you start PRs on them? I currently do not feel to comfortable with the hierarchy... We get lots of dependencies here. I am not sure how fast we find agreement on these things.